src/framework/protobuf/appearances.pb.h

```diff
@@ -0,0 +1,10533 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: appearances.proto
+
+#ifndef GOOGLE_PROTOBUF_INCLUDED_appearances_2eproto
+#define GOOGLE_PROTOBUF_INCLUDED_appearances_2eproto
+
+#include <limits>
+#include <string>
+
+#include <google/protobuf/port_def.inc>
+#if PROTOBUF_VERSION < 3018000
+#error This file was generated by a newer version of protoc which is
+#error incompatible with your Protocol Buffer headers. Please update
+#error your headers.
+#endif
+#if 3018000 < PROTOBUF_MIN_PROTOC_VERSION
+#error This file was generated by an older version of protoc which is
+#error incompatible with your Protocol Buffer headers. Please
+#error regenerate this file with a newer version of protoc.
+#endif
```

Comment:
There is also one important thing about what @lgrossi said.
These *.cc and *.h were compiled by protoc version 3.18.0, and need the exact same version in order to build the complete project.
The latest stable version of protobuf is v.3.19.4, so I had to install outdated dependencies in order to build this code.

Reply:
Even if I update these files now, it will break later, when there is update for protobuf, so that's useless.
We can remove that if check I guess.

Reply:
The good solution will be to configure build files to automatically create those files from the .proto before building.
I'm not familiar enough with cmake to make it

Reply:
same

Reply:
Then probably the most we can do is to explain in the wiki how to create the proto files.
I'm kinda afraid it could be a source of issues if we make it hard to use.